communication/message_formatting.md

```diff
@@ -4,12 +4,25 @@ Message Formatting
 - **3.5.1** ![](/badge/req.png) A client must encode emoticons as Unicode emoji.
 
 - **3.5.2** ![](/badge/req.png) A receiving client will make a best-effort to display
-  text exactly as it was sent, and must not transform the text in any way.
+  text exactly as it was intended, and must not materially change the text in any way.
     - The above rule does not apply if malformed UTF-8 text is received. In
       this case, a client may transform the text to any extent necessary to
       make it displayable.
+    - A material change is any change that likely will change the structure, layout,
+      content, meaning, text, or intent of the message.
 
-- **3.5.3** ![](/badge/req.png) A sending client must not insert or remove any bytes from
-  the original input.
+- **5.3.3** ![](/badge/rec.png) It's a strong suggestion that messaging clients support 
```

Comment:
We don't use phrases like "It's a strong suggestion". We use "may" or "shall" and the "rec" and "req" tags to denote a requirement or a recommendation. Re-word this to "A client may support a subset of the" blah blah blah, and optionally reference a table somewhere like stal suggested.